src/components/carousel/index.stories.mdx

```diff
@@ -0,0 +1,146 @@
+import { Meta, Story, Canvas, ArgsTable } from "@storybook/addon-docs";
+
+import Carousel from ".";
+
+export const SampleCard = ({ src }) => (
+	<div>
+		<h3>
+			<img style={{ padding: "0 0.25rem" }} src={src} alt="meow" />
+		</h3>
+	</div>
+);
+
+<Meta title="Components/Carousel" component={Carousel} />
+
+# Carousel
+
+A `<Carousel>` is a container for child components that may be viewed in sequence,
+forward or backward. The container can take in markup of children div containers. Here is a simple example:
+
+```jsx
+<div>
+	<h3>
+		<img
+			style={{ padding: "0 0.25rem" }}
+			src="https://loremflickr.com/400/300"
+			alt="my image 1"
+		/>
+	</h3>
+</div>
+
+<div>
+	 <h3>
+		<img
+			style={{ padding: "0 0.25rem" }}
+			src="https://loremflickr.com/400/300"
+			alt="meow"
+		/>
+	 </h3>
+</div>
+```
+
+Or you can pass in a collection of react components that act as "slides" inside the carousel. Here is a simple example:
+
+```jsx
+const SampleCard = ({ src }) => (
+	<div>
+		<h3>
+			<img style={{ padding: "0 0.25rem" }} src={src} alt="meow" />
+		</h3>
+	</div>
+);
+```
+
+The examples used below will use the `SampleCard` component for brevity.
+
+## Usage
+
+```jsx
+import { Carousel } from "@wpmedia/arc-themes-components";
+
+const Feature = () => <Carousel>Component Children</Carousel>;
+```
+
+## Properties
+
+<ArgsTable of={Carousel} />
+
+## Stories
+
+** Multiple Items**
+
+<Canvas>
+	<Story name="Multiple Items">
+		<div
+			style={{
+				marginLeft: "auto",
+				marginRight: "auto",
+				maxWidth: "90rem",
+			}}
+		>
+			<Carousel>
+				<SampleCard src="https://loremflickr.com/400/300" />
+				<SampleCard src="https://baconmockup.com/400/300" />
+				<SampleCard src="https://www.placecage.com/400/300" />
+				<SampleCard src="http://placeimg.com/400/300/any" />
+				<SampleCard src="https://loremflickr.com/400/300" />
```

Comment:
@visionbegin , @matthewroach , @JackHowa , @nschubach : Wondering if we should use mock images (showing resolution & aspect ratio) in this repo instead of using a service. Here's an example. Thoughts?
![image](https://user-images.githubusercontent.com/452134/158692224-8907ae7f-71e9-4b8c-b011-c6dcee76a991.png)

Reply:
There is an image already available to us in the storybook and is a local image - take a look at the Image component